### ISSUE TITLE
[Bug] 7.x build breaks on invalid css color

### ISSUE THREAD

**Issue by le0nik** [gpbl/react-day-picker#573]
https://github.com/gpbl/react-day-picker/blob/v7.0.1/lib/style.css#L66

Error:

```
Invalid CSS after "  color:": expected expression (e.g. 1px, bold), was "#8b9898b4;"
```

**Comment by gpbl** [comment]
https://caniuse.com/#feat=css-rrggbbaa

**Comment by gpbl** [comment]
Thanks for the report @le0nik. I've published the fix in v7.0.2 (hoping it is the last patch :)
